backend/corpora/upload_success/app.py

```diff
@@ -17,9 +18,7 @@ def success_handler(event: dict, context) -> None:
     """
     dataset_uuid = event["dataset_uuid"]
 
-    try:
-        with db_session_manager() as session:
-            success_status = {DbDatasetProcessingStatus.processing_status: ProcessingStatus.SUCCESS}
-            processing_status_updater(session, dataset_uuid, success_status)
-    except Exception:
-        logger.exception(f"Dataset {dataset_uuid}: Failed to update processing_status to SUCCESS")
+    with db_session_manager() as session:
+        dataset = Dataset.get(session, dataset_uuid)
+        success_status = {DbDatasetProcessingStatus.processing_status: ProcessingStatus.SUCCESS}
+        processing_status_updater(session, dataset.processing_status.id, success_status)
```

Comment:
ID passed to `processing_status_updater()` should have been the ID of the `dataset_processing_status` row, _NOT_ the dataset. 🤦

Reply:
good catch